gcsstore/gcsstore_test.go

```diff
@@ -0,0 +1,337 @@
+package gcsstore_test
```

Comment:
It would be beneficial to also add tests for the GCSApi as that is the part which is directly communicating with the GCS.